ext/reflection/tests/ReflectionParameter_getClassName.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Test ReflectionParameter::getClassName() usage.
+--FILE--
+<?php
+
+use Bar\Baz;
+
+class Foo {
+  public function bar(Qux $qux, $bar, Baz $baz, \Bar\Quz $quz) {}
```

Comment:
Nope! Just classes.

I'll sort this all the morning. It's 4am here in NY.

Sent from my iPhone

> On Jan 31, 2015, at 3:41 AM, Markus Staab <notifications@github.com> wrote:
> 
> In ext/reflection/tests/ReflectionParameter_getClassName.phpt:
> 
> > @@ -0,0 +1,25 @@
> > +--TEST--
> > +Test ReflectionParameter::getClassName() usage.
> > +--FILE--
> > +<?php
> > +
> > +use Bar\Baz;
> > +
> > +class Foo {
> > +  public function bar(Qux $qux, $bar, Baz $baz, \Bar\Quz $quz) {}
> Should this alsy contain array and callable typehtint?
> 
> —
> Reply to this email directly or view it on GitHub.
>